pysolr.py

```diff
@@ -191,6 +191,9 @@ def safe_urlencode(params, doseq=0):
     which can't fail down to ascii.
     """
     if IS_PY3:
+        for key, val in params.items():
```

Comment:
In general, pysolr has avoided doing type coercion to avoid needing to know what type(s) Solr supports for a given field (e.g. a parameter like `group` is documented as accepting `true` but will also accept `on` or `yes` but not `1` or `True`) but I think this is relatively safe because Solr doesn't accept that value natively so the only case where this could cause a problem is if someone was passing a Solr string which they for some reason wanted to have processed literally — for example, if I had a Python app which used a Solr StringField for something and expected the literal value `True` or `False` because that field type isn't case-insensitive.